libs/wallet/src/web3-react/hooks/useIsSmartContractWallet.ts

```diff
@@ -41,14 +41,15 @@ export function useIsSmartContractWallet(): boolean {
   const isArgentWallet = useIsArgentWallet()
   const isSmartContract = useCheckIsSmartContract()
   const isAmbireWallet = useIsAmbireWallet()
+  const isSafeWallet = useIsSafeWallet()
 
   useEffect(() => {
     if (!account) {
       setIsSmartContractWallet(false)
     } else {
-      setIsSmartContractWallet(Boolean(isAmbireWallet || isArgentWallet || isSmartContract))
+      setIsSmartContractWallet(Boolean(isSafeWallet || isAmbireWallet || isArgentWallet || isSmartContract))
```

Comment:
For some reason Safe was not included in this check.
It's covered by the smart contract check, but so are the other wallets 🤷